dkron/invoke.go

```diff
@@ -61,7 +62,7 @@ func (a *Agent) invokeJob(job *Job, execution *Execution) error {
 	execution.Success = success
 	execution.Output = output.Bytes()
 
-	rpcServer, err := a.selectServerByKey(execution.Key())
```

Comment:
can `selectServerByKey` be removed?

Reply:
according to the code, there are no occurrences.